### DISCUSSION TITLE
What crowdsec bouncer works with NPMPlus?

### DISCUSSION THREAD

**Discussion by yurividal** [ZoeyVid/NPMplus#1232] [Q&A] [answered]
I already have crowdsec working on my server, and i have the firewall bouncer working.
Probem is that I recently moved my services to a cloudflare proxy, so, the firewall bouncer is not doing much anymore to block bad actors, since they come with a cloudflare IP.
So, I want to implement the blocks on the NPM proxy. What bouncer do I use with npmPlus?
Does open openresty work?

**Comment by Zoey2936** [comment]
npmplus is your bouncer

**Reply by yurividal** [reply]
Then How do I tell it my api and key for my crowdsec?

**Reply by Zoey2936** [reply, marked as answer]
you can find an explanation it the readme: https://github.com/ZoeyVid/NPMplus#crowdsec

**Reply by Zoey2936** [reply]
did this work for you?
